.sphinx/.wordlist.txt

```diff
@@ -61,6 +63,7 @@ ECDSA
 EiB
 Eibit
 endian
+ES
```

Comment:
@ru-fu happy with those?

Personally, I'm not convinced that `ES` and `SEV` are really suitable for inclusion.
And I'd prefer we add `Diffie-Hellman` rather than have it as two words, especially as I believe it is meant to be written with a hyphen in the first place.

Reply:
If we want to introduce the abbreviations SEV and SEV-ES, we need to include them here or the spellchecker will complain. And since we're using them in the option names and the messages, I think we should introduce them.

For the hyphen, that's a limitation of the spellchecker - it treats a hyphen as a separator. Which means it will be looking for Diffie and Hellman separately. :(

So yes, happy with the update. :)